data/applications.yml

```diff
@@ -2,230 +2,230 @@
 
 - github_repo_name: collections-publisher
   type: Publishing apps
-  team: "#taxonomy-and-nav"
+  team: "#govuk-taxonomy"
```

Comment:
should this be #govuk-content-pages ?

Reply:
Potentially - feel free to change it since I don't have context of who owns what apart from platform health owns nearly everything